src/components/PostInfo/PostInfo.jsx

```diff
@@ -2,0 +1,21 @@
+import './PostInfo.scss';
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} />
```

Comment:
Ensure that the `UserInfo` component is designed to accept a `user` prop and render it correctly. The `post.user` should be an object with the necessary user details.